authorize/run.go

```diff
@@ -105,10 +105,10 @@ func (a *Authorize) runDataTypeSyncer(ctx context.Context, typeURL string) error
 		}
 
 		serverVersion = res.GetServerVersion()
-		recordVersion = res.GetRecordVersion()
 
 		for _, record := range res.GetRecords() {
 			a.updateRecord(record)
+			recordVersion = record.GetVersion()
```

Comment:
nit: might be helpful to have declaration of `recordVersion` closer to its initialization

Reply:
Not sure I follow. Where should I put the declaration?